.github/workflows/release.yaml

```diff
@@ -33,6 +33,6 @@ jobs:
           helm repo add incubator https://kubernetes-charts-incubator.storage.googleapis.com/
 
       - name: Run chart-releaser
-        uses: helm/chart-releaser-action@v1.0.0-alpha.1
+        uses: helm/chart-releaser-action@v1.0.0-alpha.2
```

Comment:
Should this be `alpha.3`?

Reply:
for this, the `alpha.2` is the latest one, as we can see here: https://github.com/helm/chart-releaser-action/releases